File: rb/test/exists_test.rb
# FreeToPlayGames SDK exists test

require "minitest/autorun"
require_relative "../FreeToPlayGames_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = FreeToPlayGamesSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { FreeToPlayGamesSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await FreeToPlayGamesSDK.test()
    equal(null !== testsdk, true)
  })

})
